packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -761,6 +761,19 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   setUpLocusMediaSharesListener() {
     this.locusInfo.on(EVENTS.LOCUS_INFO_UPDATE_MEDIA_SHARES, (payload) => {
+      const {contentId, disposition} = payload.current;
+
+      // Check if screenshare is being taken over by a remote
+      // If so, terminate the share and allow the remote to be on the floor
+      // Terminating the share allows the user to screenshare again without any issues
+      if (
+        this.isSharing &&
+        this.selfId !== contentId &&
+        disposition !== FLOOR_ACTION.RELEASED
+      ) {
+        this.updateShare({sendShare: false, receiveShare: true});
```

Comment:
Whenever I try this, I get an error of `'The peer connection is currently negotiation an offer. Please wait for a few seconds and try again.'`. Should we first manually set `this.isSharing = false` and then wait for the media to be ready to stop the share? 

@arun3528 also mentioned that the other clients don't actually stop the share, just pause it, when the floor gets released.

Reply:
receiveShare should come from the media properties per @arun3528